config/2gig/ct32.xml

```diff
@@ -155,13 +155,17 @@
       <Item label="Disabled" value="0"/>
       <Item label="Enabled" value="1"/>
     </Value>
-    <Value genre="config" index="7" label="Thermostat Swing Temperature" max="8" min="1" size="1" type="list" units="" value="0">
+    <Value genre="config" index="7" label="Thermostat Swing Temperature" max="8" min="1" size="1" type="list" units="" value="2">
       <Help>
-        The Auxiliary/Emergency configuration command enables or disables auxiliary/emergency heating in the thermostat. Auxiliary/emergency heating is only available if the thermostat is configured in heat pump mode and with at least one stage of auxiliary heating. This command enables auxiliary / emergency heating when the thermostat is in Auto mode. The Thermostat Set Mode command with mode Auxiliary/Emergency Heat will enable emergency heating but only if the thermostat is in Heat
-        mode. This command should only be used on thermsotats that support Auxiliary/Emergency Heat thermostat mode.
+        The Temperate Swing (HVAC cycling rate) is the desired variance in temperature between the thermostat setting and the room temperature required before the heating or cooling system will turn on.
       </Help>
-      <Item label="0.05F" value="1"/>
-      <Item label="0.1F" value="2"/>
+      <Item label="0.5F" value="1"/>
+      <Item label="1.0F" value="2"/>
+      <Item label="1.5F" value="3"/>
+      <Item label="2.0F" value="4"/>
+      <Item label="2.5F" value="5"/>
+      <Item label="3.0F" value="6"/>
+      <Item label="3.5F" value="7"/>
       <Item label="4.0F" value="8"/>
     </Value>
     <Value genre="config" index="8" label="Thermostat Differential Temperature" max="32767" min="2" size="2" type="list" units="F" value="4">
```

Comment:
This parameter has different data for the Set and Report command definitions. The Set takes a single differential mode (heat or cool) and a differential. The Report returns both temperatures. Doesn't look like there is any way to actually represent that in OZW.

Reply:
Since I can't add comments on non-modified lines, the Parameter 10 below is a mess. It has two entries. Looks like OZW chooses the first. The first is the Report format. It returns two bytes, while the Set takes 4 bytes. Not sure how I would fix that one.

Reply:
Seriously what? thats braindead :(

No way to handle it. 

Different sizes based on Set/Report is against the specifications. (it doesnt make it).